emitter/src/contract.rs

```diff
@@ -57,6 +57,8 @@ impl Emitter for EmitterContract {
 
         storage::set_backstop(&e, &backstop);
         storage::set_blend_id(&e, &blnd_token_id);
+        storage::set_last_fork(&e, e.ledger().sequence() - 777600); // We set the block 45 days in the past to allow for an immediate initial drop
```

Comment:
Can we set the last fork to 0 here? This requires the chain to be at least 45 days old, making the protocol very hard to deploy and use on anything but mainnet.

Further, I think it's much clearer that the initial deployment isn't a "fork", since it doesn't set the last fork time.

Reply:
Un-resolving this as I believe it was missed